Source/Core/DolphinQt2/GameList/GameList.cpp

```diff
@@ -128,12 +134,19 @@ void GameList::MakeListView()
                           !SConfig::GetInstance().m_showFileNameColumn);
 
   m_list->verticalHeader()->hide();
+  m_list->horizontalScrollBar()->setHidden(true);
```

Comment:
You should be using `setHorizontalScrollBarPolicy` here instead, as simply hiding the scrollbar just makes an empty gray bar appear.